packages/core/src/domain/tracekit/tracekit.ts

```diff
@@ -77,7 +78,18 @@ function tryToParseMessage(messageObj: unknown) {
   let name
   let message
   if ({}.toString.call(messageObj) === '[object String]') {
-    ;[, name, message] = ERROR_TYPES_RE.exec(messageObj as string)!
+    try {
+      ;[, name, message] = ERROR_TYPES_RE.exec(messageObj as string)!
+    } catch (error) {
+      throw new Error(
+        `Tracekit try parse error: ${String(error)} ${jsonStringify({
+          messageObj,
+          // eslint-disable-next-line @typescript-eslint/unbound-method
+          exec: String(RegExp.prototype.exec),
+          ERROR_TYPES_RE,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏JSON.stringify a regex shows `"{}"`, maybe toString it?

```suggestion
          ERROR_TYPES_RE: String(ERROR_TYPES_RE),
```

Reply:
Yes, thanks